src/sentry/sentry_metrics/metrics_interface.py

```diff
@@ -0,0 +1,42 @@
+import abc
+from typing import Mapping
+
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
+
+
+class GenericMetricsBackend(metaclass=abc.ABCMeta):
```

Comment:
Is there a reason we need to use metaclasses here rather than just regular abstract base classes? I think it adds complexity that might not be required

Reply:
Switched it to the regular ABC